dataverse-persistence/src/main/resources/Bundle_pl.properties

```diff
@@ -1523,7 +1523,7 @@ file.download.subset.header=Pobierz podzbi\u00F3r danych
 file.preview=Podgl\u0105d:
 file.previewMap=Podgl\u0105d mapy:
 file.fileName=Nazwa pliku
-file.type.tabularData=Dane tabelaryczne
+file.type.tabular_data=Dane tabelaryczne
```

Comment:
Not sure why we changed label name. You can't possibly reach this label by facets mechanism.